webcompat/static/js/lib/bugform.js

```diff
@@ -467,6 +470,17 @@ function BugForm() {
     }, this));
   };
 
+  // See function autoExpand in issues.js
+  this.autoExpand = function() {
+    $(document).one("focus.js-autoExpand", "textarea.js-autoExpand",function(){})
+    .on("input.js-autoExpand", "textarea.js-autoExpand", function() {
+      this.style.overflow = 'hidden';
+        this.style.height = 0;
```

Comment:
So, looking at this again, I'm totally wrong. Sorry!

> Once we delete the long comment, the height should decrease too. right? Probably, for that we will need to set height =0

So, we do want to set height to 0 (or some value like the original offsetHeight of the textarea element). The reason why is we need to re-caculate the height since we've deleted some stuff now. 

So sorry for sending you down a confusing rabbit hole there!